manifest.json

```diff
@@ -1,27 +1,28 @@
 {
-
-  "manifest_version": 2,
-  "name": "wxIF",
-  "version": "0.2",
+  "applications": {
+    "gecko": {
+      "id": "wxif@morbo.org",
+      "strict_min_version": "48.0"
```

Comment:
What's the point of this? What value does it add?

Reply:
This adds two benefits:
1. Without `strict_min_version` set, FF defaults to `42a1` which makes it (theoretically) possible to install it on non-WebExt versions, hence my PR comment about it:
   > Sets minimum version to FF48 (introduced WebExt API)
2. Adding `id`, together with a (hopefully non-existent) "mail address", enables easier recognition within FF's profile folder: the current XPI filename `{8b3ee44a-0805-4c2c-acef-15aab34fbd20}.xpi` is changed to `wxif@morbo.org.xpi` - just like other WebExtensions already do: `uBlock0@raymondhill.net.xpi`, `https-everywhere-eff-bill@eff.org.xpi`, etc.
  See MDN on [Extension ID format](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/applications#Extension_ID_format) for further details.

Reply:
Additionally, [MDN also states](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/WebExtensions_and_the_Add-on_ID#When_do_you_need_an_add-on_ID) that a constant ID is required if sync-support is desired.

For implementing #4, I'm aiming for such support (see [my comment](https://github.com/gcp/wxif/issues/4#issuecomment-359801116) there).

Reply:
>FF defaults to 42a1 which makes it (theoretically) possible to install it on non-WebExt versions,

I'm not sure what you mean by "FF defaults it" but AMO clearly recognizes and serves it to FF57+ only.

>Additionally, MDN also states that a constant ID is required if sync-support is desired.

We don't use sync support now, and syncing the position of the window seems like a bad idea (it is *not* something you want to carry over between machines!). We can add this when we have a use for it, and we do not now.

Reply:
I don't want to implement syncing of the window position, but instead set options (e. g. maps provider, display units of lat/long, etc.).

Reply:
MDN [states](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/manifest.json/applications) that omitting `strict_min_version` results in AMO setting it to `42a1`:
> `strict_min_version`: minimum version of Gecko to support. Versions containing a "*" are not valid in this field. **Defaults to "42a1".**